doc/content/msr/msre/multiphysics_rz_model/msre_multiphysics_core_model.md

```diff
@@ -0,0 +1,326 @@
+# Molten Salt Reactor Experiment (MSRE) Multiphysics Model
+
+*Contact: Mauricio Tano, mauricio.tanoretamales\@inl.gov*
+
+*Model summarized and documented by Andres Fierro, Dr. Samuel Walker, and Dr. Mauricio Tano*
+
+*Model link: [Griffin-Pronghorn Steady-State Model](https://github.com/idaholab/virtual_test_bed/tree/devel/msr/msre/multiphysics_core_model/steady_state)*
+
+!tag name=MSRE Griffin-Pronghorn Steady State Model pairs=reactor_type:MSR
+                       reactor:MSRE
+                       geometry:core
+                       simulation_type:core_multiphysics
+                       input_features:multiapps
+                       code_used:BlueCrab
+                       computing_needs:Workstation
+                       fiscal_year:2023
+
+This model of the MSRE utilizes MOOSE to create a 2D, RZ (cylindrical coordinates) mesh of the MSRE.
+Griffin computes neutronics and resulting normalized power source [!citep](Javi23).
+Pronghorn performs medium-fidelity, coarse mesh thermal-hydraulics analysis of the core, upper plenum, pump, downcomer, and lower plenum [!citep](mau23).
+Griffin and Pronghorn are coupled via the `MultiApp` system.
+The parts of the MSRE loop are represented in [MSRE_pgh_mesh_blocks].
+The model is an axisymmetric model with the left vertical axis being the axis of symmetry.
+
+The fuel salt flows down the `Downcomer`, through the `Lower Plenum`, and up into the `Core`; then, it is collected at the `Upper Plenum`, and, finally, it is circulated through the `Pump` into the loop again.
+A porous medium approach is used to model flow conditions in the core with a vertical porosity of `0.22283`.
+No rugosity is assumed when computing the friction factors.
+An anisotropic friction source coefficient keeps the flow approximately 1-Dimensional at the core, to reproduce the fluid behaviors in the core channels.
+
+!media msr/msre/MSRE_pgh_mesh_blocks.png
+       style=width:80%;margin-left:auto;margin-right:auto
+       id=MSRE_pgh_mesh_blocks
+       caption=Subdomain (left) and mesh (right) of the multiphysics model.
+
```

Comment:
Please insert a paragraph here about creating the mesh using the mesh.i file
Jun asked for it there https://github.com/idaholab/virtual_test_bed/issues/311#issuecomment-1875969316

Reply:
done